js/views/callinfoview.js

```diff
@@ -284,6 +301,9 @@
 				}.bind(this)
 			});
 
+			this.ui.passwordOption.hide();
+			this.ui.passwordButton.show();
```

Comment:
It would be better to move this into the `success` function.

Also, if there is an error in the `ajax` call it should be handled by letting the user know somehow (but without hiding the input field).